File: ympa_fsharp_functions_server/src/model/OfferCampaignStatusDTO.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.OfferCampaignStatusType

module OfferCampaignStatusDTO =

  //#region OfferCampaignStatusDTO

  [<CLIMutable>]
  type OfferCampaignStatusDTO = {
    [<JsonProperty(PropertyName = "campaignId")>]
    CampaignId : int64;
    [<JsonProperty(PropertyName = "status")>]
    Status : OfferCampaignStatusType;
  }

  //#endregion

File: src/main/scala/org/openapitools/server/model/OfferCampaignStatusDTO.scala
package org.openapitools.server.model


/**
 * Статус товара в магазине.
 *
 * @param campaignId Идентификатор кампании.  for example: ''null''
 * @param status  for example: ''null''
*/
final case class OfferCampaignStatusDTO (
  campaignId: Long,
  status: OfferCampaignStatusType
)

